04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+The population change can be dynamic depending on various environmental factors, fetched from the rest of the system. For example, a Disaster could supposedly cause the population density of deer to halve, making it harder to forage and therefore, causing lower returns. For future implementations, the team uses a single species population model and more specifically logistic modelling for the deer and fish population. The model can be described in the equation below.\\
+
+\begin{equation}
+\frac{\mathrm{d} P}{\mathrm{~d} t}=k(N-P)
+\end{equation}
+
+\begin{itemize}
+        \item “P” is the total Population \item “N” is the maximum deer/fish population (“carrying capacity” formally)
+        \item “k” is the growth coefficient
+        \item “t” is time
+    \end{itemize}
+
+In our implementation \textbf{“k”} and \textbf{“N”} are constants but it would be interesting to see what would happen if we were to increase the growth coefficient, causing a population growth and observing the islands’ dynamics. Will they keep on “investing” higher resources each turn, taking advantage of the deer/fish abundance, or will they stick to their original, safe and static strategy?\\
+
+Below, we can see a simulation of the implemented logistic model, with a growth coefficient of 0.2 and \textbf{“N”} of 8. It can be seen that overtime there is an unpredictable pattern to population changes, which would be interesting to see affecting the tier system and in extend, the deer/fish foraging strategies of the agents.\\
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/Images/Deer population over time.PNG}
+    \caption{Deer population over time (turns) under Logistic modelling with a growth coefficient of 0.2.}
+    \label{Images:Deer population over time}
+\end{figure}
+
+Modelling the population (or linking it to other environmental elements) allows us to make sure that the system’s dynamics are varying, making it harder for islands to settle onto a single foraging method because it always results in better returns. Pareto optimality is therefore harder to achieve and thus maximising social welfare is trickier, requiring agents to make more informed decisions, recognizing possible patterns. Of course, that would be an easier task for the islands, if they had knowledge on what is affecting the population (i.e. disasters), whereas a logistic model function might be more abstract in the eyes of our agents, requiring more processing. \\
+
+\subsubsection{ Incorporated Changes: Method of resource allocation}
```

Comment:
hm.. have **a consistent style about capital letters in the title**s..
suggestion:
\subsubsection{ Incorporated Changes: Method of Resource Allocation}

Reply:
agree and this style with title capitalisation should be consistent across the doc not just this section too

Reply:
excellent point and explanation on 175

Reply:
Thanks Yiannis😁